README.md

```diff
@@ -1,13 +1,24 @@
-# Kubeflow Anywhere
+# Kubeflow
 
-The Kubeflow project is a new open source Github repo dedicated to reducing friction in doing Machine Learning on Kubernetes. Contained in this repository are:
+The Kubeflow project is dedicated to making Machine Learning on Kubernetes easy, portable and scalable. Contained in this repository are:
```

Comment:
The elements listed below aren't included in the repository; they are included in the Kubeflow distribution. Should we change the text to something like
Contained in the Kubeflow distribution? or
Contained in the Kubeflow stack?

Reply:
+1.